### ISSUE TITLE
gaster or ipwndfu

### ISSUE THREAD

**Issue by hatvce** [LukeZGD/Legacy-iOS-Kit#324]
can i use gaster dfu instead use ipwndfu to downgrage iphone 5s

**Comment by LukeZGD** [comment]
yes, sure. legacy ios kit does provide choices for which pwndfu tool if supported
